tests/ios/Podfile.lock

```diff
@@ -517,7 +517,7 @@ PODS:
   - RNFBApp (8.2.0):
     - Firebase/CoreOnly (= 6.27.0)
```

Comment:
while in here, might as well go to 6.28.1 - not trying to be aggressive, just that 6.28.1 actually works great in my experience and as long as you're in the test app anyway it's like a "free" chance to qualify that it works assuming my experience (100% working) matches your result if you try it

Reply:
done. Built locally, see how it plays out with the CI!